Closes #2872 #2923 fix_case_when: fix case_when() and left_join() usage (#2924)
pharmaverse/admiral: NEWS.md

- `slice_derivation()` was updated such that it no longer fails if `args = NULL`
is specified. (#2875)

- `derive_summary_records()` issues now an error if a non summary function is

> used in `set_values_to` as this results in multiple records per by group. (#2872)

Reviewer: ```suggestion
used in `set_values_to` which results in multiple records per by group. (#2872)
```

- A link to the [{admiral} ecosystem](https://pharmaverse.org/e2eclinical/adam/) page was added to the README sidebar and main text. (#2881)
- The ADSL template and vignette were updated to add derivation of analysis age (`AAGE`/`AAGEU`) using `derive_vars_aage()`. This includes deriving birth date (`BRTHDT`) from birth date character variable (`BRTHDTC`) using `derive_vars_dt()`. (#2584)
- Standardized variable notation across documentation to use `--` for SDTM variables (e.g., `--DTC`) and `*` for ADaM variables (e.g., `*DTM`, `*DT`). (#2757)

> - For `derive_vars_dt()`, clarify `highest_imputation` definitions and expand date examples. (#2841)

Reviewer: or you just moved to a more sensible location?
